types/jwplayer/index.d.ts

```diff
@@ -317,6 +317,329 @@ declare namespace jwplayer {
     type NoParamEvent = 'adBlock' | 'beforeComplete' | 'complete' | 'beforePlay' | 'displayClick' | 'playlistComplete'
             | 'seeked' | 'remove';
 
+    type RequiredConfiguration = {
+        readonly file: string;
```

Comment:
I think your use of `readonly` is excessive and will make the actual typings hard to use, please remove those.

Reply:
None of these values are written to by JWPlayer, so they should be `readonly`.